Generator/DocumentGenerator.php

```diff
@@ -81,6 +82,11 @@ public function __construct()
 <fields>
 }';
 
+    /**
+     * @var Inflector
+     */
+    private $inflector;
```

Comment:
is this set anywhere? 🤔

Reply:
okay you are setting it in the `getInflector` to only trigger the deprecation once. is okay for me 👍